source/js/buyers-guide/components/social-share/social-share.jsx

```diff
@@ -1,33 +1,49 @@
 import React from 'react';
+import ReactGA from 'react-ga';
+import DNT from '../../dnt.js';
 
 const SocialShareLink = (props) => {
   let classes = `social-button`;
   let srLabel = ``;
   let link = `PrivacyNotINcluded.org`;
   let shareText = `I think ${props.productName} is ${props.creepType.toUpperCase()}. What do you think? Check out the Creep-O-Meter over on @mozilla’s ${link} holiday buyer’s guide.`;
-
-  switch (props.type) {
-    case 'facebook':
+  let shareEvent = {
+    category: `product`,
+    action: `share tap`,
+    label: `share vote `,
+    transport: `beacon`
+  };
+
+  if (props.type === `facebook`) {
       classes += " social-button-fb";
       srLabel = `Facebook`;
+      shareEvent.label += `to facebook`;
       link = `https://www.facebook.com/sharer/sharer.php?u=https://${link}`;
+  }
 
-      break;
-    case 'twitter':
+  if (props.type === `twitter`) {
       classes += " social-button-twitter";
       srLabel = `Twitter`;
+      shareEvent.label += `to twitter`;
       link = `https://twitter.com/intent/tweet?text=${encodeURIComponent(shareText)}`;
+  }
 
-      break;
-    case 'email':
+  if (props.type === `email`) {
       classes += " social-button-email";
       srLabel = `Email`;
+      shareEvent.label += `via email`;
       link = `mailto:?&body=${encodeURIComponent(shareText)}`;
+  }
+
+  let trackShareAction = () => {};
 
-      break;
+  if (DNT.allowTracking) {
+    trackShareAction = evt => {
+      ReactGA.event(shareEvent);
+    };
   }
 
-  return <a className={classes} href={link}><span class="sr-only">{srLabel}</span></a>;
+  return <a target="_blank" className={classes} href={link} onClick={trackShareAction}><span class="sr-only">{srLabel}</span></a>;
```

Comment:
due to the `target="_blank"`, we should be getting reliable GA on this

Reply:
I'm not seeing `target="_blank"`on the Facebook share button in the header, but I am seeing it on the Facebook share on the product page. Are they using different code paths?

Reply:
Yeah, `social.html` is used in the header.

Reply:
Also the reason we're adding _blank only for the vote share is because if you share without it, you can't go "back" to your vote results, so you can't only ever share to a single service, unlike the nav share links, which lets you get back to where you were with the back button.